liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/BasicSupport.java

```diff
@@ -374,4 +390,56 @@ protected String stripVersionFromName(String name, String version) {
             return name;
         }
     }
+    
+    protected void installLicense() throws MojoExecutionException, IOException {
+        if (licenseArtifact != null) {
+            Artifact license = getArtifact(licenseArtifact);
+            if (!hasSameLicense(license)) {
+                log.info(MessageFormat.format(messages.getString("info.install.license"), licenseArtifact.getArtifactId()));
+                Java installLicenseTask = (Java) ant.createTask("java");
+                installLicenseTask.setJar(license.getFile());
+                Argument args = installLicenseTask.createArg();
+                args.setLine("--acceptLicense " + assemblyInstallDirectory.getCanonicalPath());
+                installLicenseTask.setTimeout(30000L);
```

Comment:
I think 30 sec should be more than enough to replace the license files and update the WebSphereApplicationServer.properties file and the overhead to fork a new JVM.